src/platform/silabs/platformAbstraction/WiseMcuSpam.cpp

```diff
@@ -143,21 +143,26 @@ void sl_button_on_change(uint8_t btn, uint8_t btnAction)
     // Currently the btn0 is pull-up resistor due to which is sends a release event on every wakeup
     if (btn == SL_BUTTON_BTN0_NUMBER)
     {
-        if (btnAction == BUTTON_PRESSED)
+        if ((btnAction == BUTTON_PRESSED) && (sl_btn0_pressed == false))
         {
-            btn0_pressed = true;
+            sl_btn0_pressed = true;
```

Comment:
what about btn1 or any other button that would be on a customer board. Are they non-responsive for icd apps?

Reply:
BTN 1 is not in the UULP GPIO domain, so out of the box it will not work. There are only 4 GPIOs in the UULP domain, one of them is BTN 0. Any other BTN would have to be configured to one of the 4 GPIOs to work. To my knowledge, for customers we are already specifying this.

Reply:
Adding BTN1 would require additional hardware changes. I believe we can take that up in an upcoming PR.